### ISSUE TITLE
rename "smack-my-bitch-up"

### ISSUE THREAD

**Issue by MadMub** [NARKOZ/hacker-scripts#14]
Hate to be an SJW, but rename that script, I think in the spirit of an open source COMMUNITY, we don't need to name files like that. Just call is like "tell-wife-gunna-be-late" or something like that.

**Comment by FreekingDean** [comment]
Can we close this as `not getting fixed`?

**Comment by radu1337** [comment]
I second that. Creativity should not be censored, "dirty" words should not scare anyone.

**Comment by MadMub** [comment]
The issue wasn't really regarding offensive words, it was really just to avoid controversy for that particular title.

I do advocate artistic expression, so for those reasons, feel free the close the issue, apologies for any over-reaction or misinterpretation.

**Comment by anowell** [comment]
+1 for the issue (even if it's been closed). I don't think censoring "dirty" words should be the aim - if the project wants to keep `fucking-coffee.sh` in the name of free expression, by all means... But IMO, `smack-my-bitch-up.sh` reads like classic brogrammer culture, and promoting that in the name of "creativity" seems pretty pathetic. I'm not appealing on the grounds of censorship, but on the grounds of being part of the solution - not the problem.

**Comment by mushishi78** [comment]
+1 @MadMub you should reopen. we don't need misogyny on Github

**Comment by Lartza** [comment]
Not seeing a reason here to rename. If it's been translated faithfully I don't get why it couldn't stay this way, this is historical accuracy and if it deviates from the Russian story then it's dumb to go renaming stuff.

Drawing the misogyny card is ridiculous too, I don't think anyone is supporting the implied "misogyny". I don't see why if I'd write a script called ISISftw.sh that would do some stupid shit be considered supporting anything. Would it be offensive to someone? Hell yeah. Would it make a great story like this one day? Possibly.

Don't take offense of stupid shit that has been written possibly years ago, by some possibly very drunk or hungover Slavic hacker.

**Comment by FreekingDean** [comment]
Agreed @Lartza, I think the main point of this is to show what the original "hacker" wrote. Removing the names/renaming the scripts takes away from the story completely. I mean, this repo is mostly about the narrative of that persons work life there.

**Comment by mushishi78** [comment]
The context has changed entirely. When he wrote the name, he was making a private joke that was only ever intended to be seen by him. Whereas now it's a very public joke that has been seen by tens of thousands of people. It is the source of news articles that are being shared on social media and is now representative of our industry for any casual reader who clicks the link back to the repository. The last thing developers need is to be seen as celebrating casual sexism.

Also, @Lartza referring to misogyny in quotations marks, as a "card" to be played and analogous to any arbitrary and purposefully offensive statement, are all examples of misogyny.The fact that you don't understand this is indicative that you are part of the problem.

Github is not 4chan or your obscure sub-reddit, it is an inclusive safe space for all gender identities and casual sexism is not acceptable. It is an extension of an office work environment, not a place for chauvinistic bros who exclude women to make themselves feel big. Changing the name is a small compromise for the sake of projecting an environment of inclusivity and it is not essential to what makes this repo funny in the first place.

**Comment by MadMub** [comment]
I closed the issue for one important reason, it's a meaningless repo, sure it has tons of stars, but ultimately its just a joke/sarcastic/funny/troll/not an actual repo. I'm not throwing hate, I find many of those types of repos (think five.js, ArnoldC) to be interesting, including this one. But ultimately it's not in the same context of angularjs, or the latest C++ spec. This isn't a community driven code base to be used in thousands of production environments. Even if some of the code does (like a startup for networked coffee machines...), let's hope that in that context, those developers choose appropriate naming conventions for a community maintained project (we're not even talking about vulgar words, just having good coding practices).

So let's not flame, let's just move on, I personally would never use naming like that, and that's what I believe, but that doesn't make me an authority over someone else's project. This isn't an open source project representing the work of hundreds of men and women from varied backgrounds. If you want to make a statement, do what I probably should have done from the start, fork the repository and change the names yourself.

As a community/society, we need to pick and choose our battles, there are plenty of other more popular mediums that throw hate to gender/sexual orientation/race/religion etc.. like popular music, or worse, our governments... major news outlets....

Don't let this thread turn into another war of ideals, do what github and OSS was intended for: fork, derive, evolve this into something useful to you or that you believe in. Isn't that what ALL programming languages and frameworks have EVER been, a case where a technology didn't meet expectations, so a group of passionate developers made something new.

**Comment by Lartza** [comment]
Gosh I wish every day people would have real problems to discuss. Mind my mistakes in the English language I made, couldn't bother to proofread it and I'm not a native. Thanks.

I don't see why a single news like this, where the context is clearly stated out, would label the whole industry any more sexist than any other industry, or than it already is or isn't. For the love of god, the public view of hackers is probably somewhere in the lines of the most pathetic nerd person in movies/TV shows. This ain't some evil tinfoil propaganda that turns peoples minds about computer industry, it's a goddamn story of a Russian hacker who wrote stupid scripts for himself.

I admit to having to think about a response to this paragraph for some time. Probably shouldn't have used the card remark but hey, it's on the internet now, no going back only possibly explaining better what you meant. Sorry if I didn't catch your point correctly though, couldn't grasp it totally with the language barrier. But here we go.

He could have had a "rape-my-workbro.sh" (Which ironically wouldn't probably had such a fuss around it for some reason, nobody would have even complained about any rape culture probably), or "axe-my-fucking-children.sh" (Which needs a bit of Finnish culture knowledge to understand probably) for all I care, I just tried to make a point and now my analogue is bad? You seem to draw a conclusion that my analogs were purposefully offensive? Why? I just made a script called that for funsies, just like the goddamn Slavic idiot did. This script is not a targeted offense to anyone, or any group, and you seem to think people are stupid and don't understand that. There is a difference to laughing at some stupid joke like this and understanding it, and lauging at it and AGREEING with it. My thought process to this wasn't "heh, right on you go smack that bitch real HARD", if you can believe it. And laughing at something like this doesn't mean that you'd ever SERIOUSLY write a script like this to be purposefully sexist. I can laugh at this as much as I can laugh at a disabled person looking stupid without being offensive toward that person. I probably wouldn't laugh TO that person if he couldn't laugh at himself, but does finding something funny make you part of some grand problem? Honestly I don't fucking think so. Hell, I can laugh at my self if need be, you are welcome to do so too at any point! Online or face to face anything goes.

 I would have touche'd you for the 4chan and/or reddit remarks if I browsed even one of those sites. This is not a goddamn GitHub council that wrote these scripts and represents all of GitHub. It's as if you are smearing this issue to the whole of GitHub. If a student uses his freedom of speech and taught some stupid shit on a campus, and I see universities as inclusive grounds where students and staff can be of any background, would that have to label the goddamn university, or ALL universities sexist?? NO. The conclusion a sane person should draw from this repo is "Wow, what an awful name to call your stuff" not "OH MY GOD all hackers are such dicks!!!"

**Comment by mushishi78** [comment]
@Lartza I'm not reading that wall of text. Sorry, but I don't care that much.

@MadMub I would appreciate if you re-opened the issue despite your reservations, the issue has not been resolved.

@NARKOZ It would be appreciated if you changes the file name "smack my bitch up" to something beign, as this repo's gained a lot of traction today in the wider community.

**Comment by MadMub** [comment]
I'll reopen since there's no point in someone opening a dupe, going to unsub from the issue though, the contributors/community can debate, decide, and implement  what's most appropriate.

Going forward, to @all future "trendy" interesting, "new media", experimental, based repos, choose words with pinpoint precision. Because regardless of who thinks what is right or wrong, or whether or not anyone can objectively evaluate context/intent/semantics, you will ultimately smear your work in a situation like this.

Unless of course the whole point was to spark controversy...

**Comment by ceer** [comment]
It may refer to this song by The Prodigy: https://en.wikipedia.org/wiki/Smack_My_Bitch_Up

**Comment by slang800** [comment]
@MadMub - Hate to be an SJW, but you should probably apologize to all the people you've offended by opening this issue. By suggesting that "smack-my-bitch-up" is sexist, you've actually committed a racist microaggression against minority communities that don't use your white vernacular. For those oppressed minorities, the phrase "smack my bitch up" has a totally different connotation, and you've marginalized them by saying that their language is inherently "sexist".

**Comment by cimmanon** [comment]
Just curious, are you in favor of censoring books like To Kill a Mockingbird due to their use of the N-word as well?

**Comment by Brawl345** [comment]
r u fucking serious?

**Comment by mikeminneman** [comment]
@mushishi78 You have to be kidding me. Is it offensive? Sure. Is it a joke that should be taken in context? Yes. So please calm down, if it offends you, fork it and make your own copy but by grabbing the pitchforks and trying to politically correctly whitewash everything you're showing nothing other than you have no sense of humor. 

You come in here, tell us were all insensitive jerks, and then when someone tries to explain the other side of the issue you tell him you don't have time to read his response? You're the one being insensitive and you DO NOT have the moral high ground.

Leave the repo as it is, close the issue, and go crusade somewhere that matters.

**Comment by pedroardaglio** [comment]
@mikeminneman Thanks!

**Comment by mushishi78** [comment]
@mikeminneman I'm perfectly calm, very reasonable and not really offended. This has very little to do with moral superiority or "crusading" and everything to do with being a responsible adult whose aware of ones privilege and the demographics of ones industry. If we want more women developers, it's important that the culture on Github remain professional and free from casual misogyny.

**Comment by MrBenJ** [comment]
@mushishi78 - Can you accurately speak for the entire Github community that this should be changed? From this thread alone, I feel like you're the only one who's seriously fighting for a change that's extremely miniscule. Do you represent Github? Why are you going around touting and speaking on their behalf? "It's important that the culture on Github remain professional and free from casual misogyny." 

Have you asked a woman's opinion on the issue? Most (including my many female coworkers who are laughing about this right now), would realize it's a joke and understand that nobody is really promoting violence against women. To assume otherwise and take offense on behalf of women is to offend them by treating them like they can't make their own decisions, or what's funny and what isn't.

Yes, you may be a part of the Github community but to inflate an issue like this, as you have been doing, is absolutely incredulous. Let it go. You are making yourself look like an absolute fool. 

Just let it go.

**Comment by bf4** [comment]
A screenplay in one part

Alan: whoa, he had a script called 'smack-my-bitch-up.sh'

Dave: That's an awful name. What does it do?

Alan: It seems to notifies his wife he's working late

Sarah: Wow, I thought it would be worse than that

Steve: Yeah, it's just kind of an foul name, but neat code. Let's share it.

Dave: Great idea! Should we keep the name?

Sarah: I don't know.  If we change it, we still might want to reference the source and original name

Alan: That sounds like a good idea.

Dave: Yeah, the file name may be funny to some people. Others might not like it.  But it's the code that's the fun part.  The name is just part of the story.

Brett: You all are insane! The man called his script 'smack-my-bitch-up.sh'! Who are we to change it?  Without a license, all rights are reserved!

Alan: Brett, chill, the company owns it anyhow. How about we let our boss pick the name.

Brett: But, she might change it!

Alan: So?

Brett: But it's funny.

Alan: Can't it still be a funny story, even if we rename the file.

Brett: Well, it's not what I'd do, but I guess it's fair.

**Comment by samis** [comment]
I personally do not like any form of puritanism and the issue raised here is nothing except puritanism. I see two types of people who would be offended by this: 1) People who take the joke literally. We can't really do much about this anyway so there's no reason to change here. 2. Self-righteous people who believe they have the right and capability to feel offence on the behalf of a larger group they may or may not represent. I'm not sure if it's worth it to change for this reason. 

Side note: It is issues like these that make me not want to associate with the type of liberal who follows this line of thinking. Mountains are made out of non-existent molehills. I apologize if this upsets anyone, but I have a quite strong opinion on the subject.

**Comment by tstaton56** [comment]
@MrBenJ - I'm a female engineer and I'm commenting using an account other than my real one for reasons that ought to be obvious to anyone who has spent any time as a woman on the internet. Normally I wouldn't touch a discussion like this with a ten foot pole because doing so is inviting harassment. The only reason I'm commenting is that you were asking @mushishi78 if he'd actually a woman's opinion.

Here's my opinion: when I saw that particular script name, I cringed. I understand that it's probably "just a joke" named after that song by The Prodigy, but these sorts of jokes are so, so, so tiresome and so very typical. The discourse around it is to be expected too, which is to say a whole bunch of guys stating why it's not a big deal and why we shouldn't take it seriously (and if we do, we must fall into the dreaded SWJ category and we're just hypersensitive). 

Thing is, it gets hard to not feel seriously fatigued by it when you're bombarded with this sort of stuff in the community every day. It's not even offense so much as it's just another exhausting day. It is casual sexism - full stop. You can decide whether or not the joke itself is more important than the fact that the joke is casually sexist, that's fine.

That's MY opinion, take it or leave it. I'm not going to demand anything nor am I likely to participate in this discussion beyond this comment. Cheers.

**Comment by tavrinn** [comment]
Someone needs to add a poll to github so we can vote

**Comment by tavrinn** [comment]
+1 with @FreekingDean

**Comment by Lartza** [comment]
@tavrinn We only have opinions on this, the repo owner has the say in this. Polls would probably be useful for some projects and could be used to gather opinions, but they can usually be organized among collaborators and owners of a project outside GitHub easily.

**Comment by cimmanon** [comment]
@tstaton56 And here you are, assuming everyone who falls into the "it's no big deal" crowd is a man.  Isn't that a little sexist?  It is so, so, so tiring to see women constantly painted as poor delicate creatures that will shatter if they hear a crude or tasteless joke.  Some women find it funny, just like some men find it questionable.

The original author of the scripts appears to be an equal-opportunity jerk if you ask me... unless you think kumar-asshole is an appropriate way to refer to someone?  It's OK to be a dick all you want to men, but being disrespectful to women is crossing the line?  Again, sounds pretty sexist.

As far as I'm concerned, this is just one dude's dirty laundry, and not at all representative of all members of the tech community.

**Comment by slang800** [comment]
@cimmanon makes a good point.

**Comment by alain-lf** [comment]
Well, in the spirit of an open source community, you can fork the repo and rename the file youself so that it can become part of your safe space.

**Comment by FreekingDean** [comment]
@alain-lf this is the best answer. If you want it changed, fork it and change it. Then distribute your own version as long as it follows the rules outlined by the license.

**Comment by lizTheDeveloper** [comment]
Hi, +1 for keeping original name, because it contains context about the original developer that might help us understand his code/art. Also +1 on not judging other people's code naming choices in a non-major open source project because being public doesn't mean it's part of a community. This is a story. Similar to not removing n****r references from Huckleberry Finn, this is probably not only a waste of time but kind of counts as censorship, and whitewashing history because you don't like a word.

That said I don't like it when women are referred to casually as bitches and the WIT scene is deplorably small, so I get it, but I think this is a great example of pointless argument over trivialities and tone policing.

But hey, maybe someone has a script that automagically opens an issue on any github repo containing the word "bitch", maybe it's called 'smack_the_patriarchy_up.sh'

**Comment by hsavit1** [comment]
![image](https://cloud.githubusercontent.com/assets/5156236/11390490/b35f3f02-9318-11e5-9a05-38d48f2000f3.png)

**Comment by LijieSong** [comment]
keep write  ,  i want it

**Comment by bassbastard** [comment]
As a native american on the eve of Thanksgiving, and just recently passed Columbus day, this shit cracks me up. 

Keep the name. It is accurate to the circumstance. 

You would think that white washing something that actually happened would be anathema to a group who turn to outrage when anything get's whitewashed.

**Comment by a-professional-developer** [comment]
I came across this repo and ultimately, this issue from the Github explore today email. Quite frankly, I'm disgusted.

We're programmers. We're smart-ish when we want to be. We're also human. We laugh, we joke, all that good stuff.

If one of us (he/she/xim/xir/whatever you're using) decides to write some shitty code (literally or metaphorically) and put it out there, **FUCKING LET THEM**. If you don't like it or you're triggered by it **STOP LOOKING AT IT**. It's literally that damn simple.

To those who are _incredibly_ triggered by this: stop imposing your feelings or thoughts on others simply because you don't like it. Deep down *you know* that you usually get your way by complaining. Some people don't take kindly to complainers, esp. if it's for a bullshit reason like safe spaces, which brings me to my next point.

**THE INTERNET IS NOT A SAFE SPACE.** My god I can't stress this enough.

If you _really_ were interested and focused on practicing what you preach (looking at you SJW's and other cultural marxists) you would be out putting in effort to setup some sort of github clone that *only* hosted safe, trigger free projects. But here you are, complaining about some goofy ass script name. **Lame. As. Fuck.**

**Comment by RypeDub** [comment]
You are all cunts and bitches. I'd smack all of you up.

Shut the fuck up and DELETE this ticket, don't even close it.

Fuck WOMEN, FUCK SJW, FUCKING CUNT BITCH WHORES

**Comment by RypeDub** [comment]
There is no "issue". The script works perfectly fine, the Unicode characters used are accepted by any computer system and the name of the script perfectly describes what the script does.

Are you literally mentally handicapped? You shouldn't be using a computer or the Internet.

**Comment by shiraabel** [comment]
Hello! Female in the thread now. Was I the only one who got the reference that this is a song by The Prodigy? I didn't take it offensively. He's sending it to his wife. I don't know what he's writing to his wife, but if he's thoughtful enough to send her a note explaining he's working late - my guess is it's not that bad. 

And the song happens to be brilliant.

**Comment by kilrah** [comment]
![12088394_773710972754789_2034549127707160966_n](https://cloud.githubusercontent.com/assets/6065069/11400896/616d87da-93a9-11e5-8d19-f66bf11a67e3.jpg)

**Comment by expl0ratory** [comment]
![image](https://cloud.githubusercontent.com/assets/3323168/11402280/b51b329e-935c-11e5-9487-f4b899f4c73c.png)

y'all need hobbies or jobs

**Comment by LijieSong** [comment]
Hey  Gays , I want to sleep now ,goodnight!

**Comment by jnferguson** [comment]
It's likely a reference to the prodigy song by the same name; its weird watching this because it exhibits the same thing the song and video were attempting to describe way back in the 1990s.

Before you decry "brogrammer culture" (which is valid), you should watch the video in question-- the gotcha is at the end: https://www.youtube.com/watch?v=2DPF_pWIy3w

**Comment by tavrinn** [comment]
![hahahahaha](https://cloud.githubusercontent.com/assets/2568052/11412321/9364094a-942f-11e5-96c9-512cbdfcc141.jpg)

**Comment by Dabitch** [comment]
There are a number of women in the thread now, @tstaton56 being one of them using a a throwaway account, and it seems that we are divided on the name here - there's those who know it's a prodigy song, and those who seemingly don't.

I'm not bothered by the name of the script one iota, and fail to see a reason this needs to be changed by anyone other than the original author of the script. 

![-smack-my-bitch-up- -597883](https://cloud.githubusercontent.com/assets/5349307/11415339/fc4fc8c8-9401-11e5-88a4-7c0d0467a597.gif)

**Comment by LijieSong** [comment]
I  want know where this coffee scalled?

**Comment by anowell** [comment]
thanks @tstaton56 for your voice, even if it needed to be anonymous.

I never saw censorship as the goal. I thought if I added a quick reminder of how this connects to the brogrammer culture problems that exist, then a handful of people would quickly jump in and reassert that changing the name would contribute to the solution without really taking away from the interesting narrative this repo tells. Apparantly, my thinking was naive.

I did take the time to ask a few women in tech for their thoughts on the repo and this thread (none interested in responding directly). Responses varied. Some thought the name was funny, understood the Prodigy reference, and simply thought the name should be left alone. But I also heard responses about how this is a yet-another-example of how uninviting many corners of tech culture still are, or how they've learned to stay quiet to avoid becoming the target of backlash. I heard stories where complaining about jokes about real sexual violence resulted in being labeled as a bad "culture fit". These are real problems, and on the spectrum ranging from "contributing to the problem" to "being part of the solution", I still believe `smack-my-bitch-up` (and perhaps a few comments in this thread) falls closer to the former.

But I'm on no mission to rid the Internet of offensive content. I'm not calling for its censorship. The issue has been closed and this project has decided that it values preserving a joke in its originality more than the concern that has been raised against it. I only bothered to respond in the off-chance that it spurs even a single person to revisit their own empathy - otherwise, my words here are wasted.

**Comment by Kyokou** [comment]
No, you aren't the majority. You are the [blue dots](https://www.washingtonpost.com/graphics/business/wonkblog/majority-illusion/). No, people don't have this weird perversion of "broculture" surrounding programming. That's just you guys. And no, a flavor-of-the-week article that cites a programmers' personal sh scripts where one is titled `smack-by-bitch-up` is not going to leave a lasting impression on the `tech community` (whatever the fuck that is even supposed to be).

In every goddamn thread like this that happens on Github I see the same 20~ people voicing their imaginary displeasure for internet brownie points. Every. Single. Time.

More people need to tell them to pound sand. Rather than rename `suicide` to `voluntaryExit` or changing `bro` to `br`.

**Comment by aerisdies** [comment]
Someone else's repo is not your safe space.Found something offensive in it? Press Ctrl+W and get on with your life.

Vonnegut was wrong: you people are the true enemies of freedom.

**Comment by VictorHM** [comment]
It's great to see that the thin-skinned SLW are starting to loose ground on their stupid crusades. At least, we can focus on the real issues and problems and enjoy coding again.

**Comment by vibhavp** [comment]
Changing the name would confuse people, because it's not what the name of the script original IRC conversation was. Why try to change history for the sole purpose of comfort? That's just lazy activism.

**Comment by SJWftw** [comment]
Really interesting that commenters here are so bemused by why this issue was raised.

Obviously, not all women are going to be offended about this. Not all men are going to be. Because neither genders are homogenous groups. And I guess it doesn't really matter either way if they are because a) it's ok for people to be offended by stuff and and b) this is a dumb joke on the internet.

But don't you get at all why it was raised? Really?

This guy has been heralded as a 'real hacker', right? And he's kind of playing into every stereotype about what a 'real hacker' is, including that hackers are sexist jerks. This guy keeps a script about his wife using a song title called 'Smack by bitch up'. As a little mental joke, a play on words, where he gets a silly kick out of jokily referencing his wife as a bitch. Obviously, not seriously, he'd probably never say it to her IRL, but late one night, it made him chuckle. But that's really not a big deal. I mean, even if he was a serious misogynist, he's just be one idiot.

That's not the point. The point is, the story gets shared, and a github repository gets created, and it's kind of admired in a in-joke kind of way - in a 'haha, what a jerk but man, he's a real hacker, right?' - and the point of sharing is BECAUSE it's an in-joke, a part of the culture, a you-and-I-get-this-but-most-wouldn't. Hey, we're hackers too -  because we share this, and we find this funny. 

So I'm not saying that you should change the file name. But the idea that this doesn't illustrate a 'brogrammer' culture? Really? Come on. 

For what it's worth, as a female programmer, I had the same reaction @tstaton56. Does that mean you should change the file name? Not really. Might it add to the general culture that puts women off programming? Sure. It was linked on a discussion board at my work which is 90% male - they thought it was pretty funny. I opened it up, read the story, saw the file name, sighed, thought - 'oh, another one of these' - closed it down, got on with my day. No big deal. A little bit sad.

**Comment by Dabitch** [comment]
> But don't you get at all why it was raised? Really?

Yes, really. I might simply be a much different generation from you, and perhaps that's why I don't see brogrammer culture in a 90s song reference. Language evolves, but only Stalin erased the past.

**Comment by SJWftw** [comment]
I think you may be slightly aggrandising this repository to think of it as History :) A History under the threat of a Stalin-esque dictatorial conspiracy, no less! But then I've probably aggrandised it already with my above comment.

Anyway, I don't think the name should change. And I guess you could ask - so there's a 'brogrammer' culture, so what? The majority of coders are male. They wrote and write most of the code. If a minority of women want to join in, and some find the culture a little bit ostracising, then they should adapt to the culture rather than the other way around. 

It's just there's a chance you might end up losing *some* talented female coders that way. I think that's why the issue was raised.

**Comment by Dabitch** [comment]
Didn't stop me twenty years ago I don't see why it would stop you today. Like I said, I guess i'm just from another generation.

**Comment by SJWftw** [comment]
Well I've been coding since 1993, so I would't say it's stopped me. But I'm not *all* female programmers. Also, things are pretty different today, what with open source repositories and internet chat rooms.

**Comment by Dabitch** [comment]
For anecdata comparison, I've been programming since it was called *programming*, which is yet another point about words and their meanings. This could thus be a generational gap as much as an actual language gap.

**Comment by DavidR91** [comment]
Frankly, the *responses* to this request are actually a far better example of how dire attitudes in tech are [to women], than the original text is.

**Comment by Superbest** [comment]
I agree, I think this awful, misogynistic name should be removed to avoid triggering valuable contributors from diverse backgrounds!

How can open source software ever produce innovative solutions if the community is not more open to diverse contributors?

**Comment by tavrinn** [comment]
I can't believe this is still going

**Comment by Karasiq** [comment]
Too fat

**Comment by jnferguson** [comment]
Wouldn't it be great if the assumption that it was even to his wife, or a female was challenged and it turned out the guy was gay, or had some weird thing  about texting his dog or something?

**Comment by Whalerus** [comment]
Why hasn't this been closed

**Comment by pcbro** [comment]
Why has this been closed?

**Comment by Sikian** [comment]
I totally object to this being closed. I'm not against free speech, but I do believe that it's not necessary to use this kind of titles and changing it would not hurt the author and would be positive for people which might feel attacked by it, i.e. women in this case.

**Comment by Dabitch** [comment]
I totally object to you speaking for me, @Sikian.

**Comment by Sikian** [comment]
@Dabitch I'm afraid I was not speaking for you but giving my opinion on the relationship between the name of the script and the implications on a collective.

**Comment by Dabitch** [comment]
> i.e. women in this case.

How was I excluded from your statement?

**Comment by mushishi78** [comment]
#NotAllWomen

**Comment by I2obiN** [comment]
@jnferguson truth

**Comment by Dabitch** [comment]
@mushishi78 That would be hilarious if it wasn't for the fact that all the men I see in this thread are talking about women's possible feels about the name this repo, while ignoring what I and @shiraabel and some of the other women I've seen in this thread. I find that offensive.

**Comment by ScholarMidnight** [comment]
Hello, I am a woman and this is a non-issue and this thread is stupid and this "culture of politically correct inclusiveness" that seems to be pervasive throughout Github in the past year is offensive to me.

**Comment by mushishi78** [comment]
@Dabitch I wasn't necessarily trying to be funny as much as gently pointing out through analogy that you seem to be confusing the modality of general statements.

**Comment by Dabitch** [comment]
Man comes into thread where women say "we have no issue with this", states he suggests name change as it'll be a postive for those 'might feel attacked by it; ie women*, thereby ignoring the women in this thread who have already spoken up and voiced their opinion on thismatter. His is a hypothetical that ignores women already speaking in the thread. 

If you don't see what is wrong with that, I can not help you.

**Comment by mjblack** [comment]
If this script name is making you feel unwelcome in tech then tech is not for you. Perhaps spending your days in a ball pit is more to your liking. This repo doesnt have to be your safe space. No one is making you come here. No one is making you use these scripts. And, if you honestly need to use these scripts and you find it offensive, fork it, rename it and be done with it. Demanding someone change something because your fee fees are hurt is a sign that you are still a child and that you need to grow up.

**Comment by mushishi78** [comment]
@Dabitch I think you may have lost some of the nuance expressed by the other women in this thread. To say they have "no issue" is probably misrepresenting them.

**Comment by tavrinn** [comment]
![hahahahaha](https://cloud.githubusercontent.com/assets/2568052/11616133/2bac62a6-9cc8-11e5-90a2-4e8a8481f57c.jpg)

**Comment by Dabitch** [comment]
@mushishi78 I'm sure they can correct me themselves, if I have misinterpreted their nuance.

**Comment by SJWftw** [comment]
@mjblack "If this script name is making you feel unwelcome in tech then tech is not for you." - why do  you think that? isn't that a little presumptuous - you get to decide that, do you? is that just open source tech that's not for them? or all tech, everywhere?
"Demanding someone change something because your fee fees are hurt is a sign that you are still a child and that you need to grow up." - writing something that condescending would make me think maturity may not be one of your real strengths. Out of interest, is there anything that someone could name a script that you'd think beyond the pale? Or is it 100% anything goes? Is that just on github, or all media/public spaces? Same if someone said it in real life? If i'm sexist/racist/homophobic to a stranger in the street, for instance, would you object or tell them 'get over it - the street isn't your safe space'? Would I at least deserve to get called out as sexist/racist/homophobic, or would that impinge on my right to free speech? How about in the workplace? (github is a part of many people's workplaces)
@tavrinn I didn't think that gif could get any funnier - i'm so glad you shared it twice.
@dabitch are you saying men shouldn't post here? or only men who agree with you should be allowed to post here? those men who disagree with you - as they don't get your viewpoint, are you saying they shouldn't speak about sexism, as that's speaking for you? but what about all the men here who do agree with you, but who don't get the viewpoint of the women who disagree with you - shouldn't you object to them posting too? so that means only women be allowed to post here? I'm so confused. One moment this thread is pro freedom of speech, another moment it's against it...

**Comment by Dabitch** [comment]
@SJWftw hey sock puppet, you're not confused at all. You're deliberately misrepresenting what I said in order to confuse things. As far as I can tell, the women in this thread are here voicing their own opinions about the changing of the name. One person here is arguing "I have no problem but some hypothetical women might have", to which I objected, as I and several other women are in this thread are already speaking up. Do I need to get a man in here to explain mine and other women's stance before it's heard?

**Comment by SJWftw** [comment]
Some women here are saying they have a problem with it. Some are saying they don't. Some men are saying they do. Some are saying they don't. What's wrong with that?

**Comment by Dabitch** [comment]
I don't actually see any real women in this thread saying they have an issue with it. I see sock puppets.

**Comment by SJWftw** [comment]
Right - so it's the second option - only people who agree with you should be allowed to post here.

**Comment by Dabitch** [comment]
@SJWftw I'm observing that no woman (with a posting history) in this thread has objected to the name.

**Comment by SJWftw** [comment]
Ah, I see. @Dabitch  - Please see @tstaton56 comment above, to explain why that is. "I'm a female engineer and I'm commenting using an account other than my real one for reasons that ought to be obvious to anyone who has spent any time as a woman on the internet. Normally I wouldn't touch a discussion like this with a ten foot pole because doing so is inviting harassment. The only reason I'm commenting is that you were asking @mushishi78 if he'd actually asked a woman's opinion."

That was the same reason for me, at least. I do this as my job - not just a hobby. I guess maybe we should be braver, but hopefully the fear of harassment shouldn't invalidate our views completely.

**Comment by tavrinn** [comment]
@SJWftw  happy to share it again seeing as you were that amused by it :D
![hahahahaha](https://cloud.githubusercontent.com/assets/2568052/11618524/206a556a-9cf0-11e5-8117-5bb08e75d3b0.jpg)

**Comment by tavrinn** [comment]
You don't have to thank me, and yes you're welcome :+1:

**Comment by artur-augustyniak** [comment]
It started with Atlassian  puking with rainbow into my terminal then was master/slave shitstorm, now this. I am horribly tired with people not doing things that should be their main activity. As for this thread. You can be female, gay or trans something developer but you can't tell me that I can't make jokes or hate you as much as i want(of course it works both ways). As far as i do not attack you physically, I can.

**Comment by SimonJiha** [comment]
SJW activists, **f**k off** with your political correctness. Just **f**k off**. Some lynch crusading jerks wanted to throw out Opal's top maintainer for his personal thoughts, now u wanna change the name of a single script. Please, go eat ur own s**t, bros. U come here and start to tell so enourmous piles of BS to the point that it becomes not even funny anymore. While u fight for 'freedom', u can't see that u urselves are the biggest enemies of freedom - u r the nazis of 21st century, with ur ostracism and ur total and utter will to punish other people for their 'thought crimes' if their opinion appears to be different from urs.

**Comment by bettibros22** [comment]
I have leaved with my husband for 10 years now and have never had access to his phone. He hides his phone from me all the time but during that period I was not bothered but  it got to a time I felt cheated on cause he always do things that makes me suspect him all the time but after talking to jeajamhacker@gmail.com for help in less than 24 hours I got help. This hacker hacked his phone and I was able to receive all my husband text messages and calls right on my phone, I also got deleted text messages from the past 4 years and you won’t believe this, my husband has a daughter outside and he never talked about it to me since we got married. Well I want to say a big Thank you to this hacker and I promised him if what I paid for works I will go round the internet to broadcast him of his good work and here I am. Jeajamhacker@gmail.com is the best
